assets/dev/js/editor/components/template-library/views/library-layout.js

```diff
@@ -12,6 +12,11 @@ module.exports = elementorModules.common.views.modal.Layout.extend( {
 	getModalOptions() {
 		return {
 			id: 'elementor-template-library-modal',
+			hide: {
+				onOutsideClick: windows?.elementor?.config?.document?.panel?.allow_closing_remote_library ?? true,
+				onBackgroundClick: windows?.elementor?.config?.document?.panel?.allow_closing_remote_library ?? true,
+				onEscKeyPress: windows?.elementor?.config?.document?.panel?.allow_closing_remote_library ?? true,
```

Comment:
```suggestion
				onOutsideClick: window?.elementor?.config?.document?.panel?.allow_closing_remote_library ?? true,
				onBackgroundClick: window?.elementor?.config?.document?.panel?.allow_closing_remote_library ?? true,
				onEscKeyPress: window?.elementor?.config?.document?.panel?.allow_closing_remote_library ?? true,
```
Right? If it was a mistake, can you please test everything again, because not sure how it worked :)

Reply:
Cache it in a variable.

```
const allowModalClose = window?.elementor?.config?.document?.panel?.allow_closing_remote_library ?? true;

hide: {
	onOutsideClick: allowModalClose,
	onBackgroundClick: allowModalClose,
	onEscKeyPress: allowModalClose,
},
```

Reply:
@louiswol94 😶‍🌫️ that was a typo I added after making sure it was working. Fixed now.